packages/@react-stately/layout/src/TableLayout.ts

```diff
@@ -439,4 +441,22 @@ export class TableLayout<T> extends ListLayout<T> {
 
     return res;
   }
+
+  // Checks if Chrome version is 105 or greater
+  private checkChrome105() {
```

Comment:
This feels like it should belong in util, not that anything else will use this.

Reply:
It's pretty specific and temporary so I kept it in here. Also this means we don't have to pull in another dep if this were in aria/utils